requirements.txt

```
@@ -0,0 +1,4 @@
+django==2.2
```

Might be better to use `~=` to get compatible security releases for Django 2.2.
```suggestion
Django~=2.2
```
(This change currently results in Django 2.2.17 being installed.)